docs/core/component.md

```diff
@@ -508,8 +508,8 @@ AFRAME.registerComponent('c', {});
 
 [sound]: ../components/sound.md
 
-`multiple` allows for a component to have multiple instances. By default, since
-`multiple` is set to `false`, a component could have one instance. For
+`multiple` allows for a component to have multiple instances per entities. By default, since
+`multiple` is set to `false`, entities could have only one instance of a component . For
```

Comment:
What about? `multiple is set to false, by default an entity can only have a single instance of the component`